README.md

```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
 ./react-bootstrap/run.sh -l
 ```
 
+## Bootstrap options
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean project with the most used libraries of React
+
+- ### Customized
+
+Choose your preferred libraries for your project and create a new React project with standards components, default screens, api services and common utils functions.
+
+###### Components
+- Checkbox
+- InputLabel
+- RadioGroup
+- Routes
+- SearchBar
+- Spinner
+- Suspense 
+- TextArea
+
+###### Screens
+- Login
+- Dashboard
+
+## Bootstrap libraries
+
+|||
+|----|-----------|
+|[react]()| v16.6.3
+
+
+- ### Clean
+
+|Name|Description|
+|----|-----------|
+|[eslint-config-wolox](https://github.com/Wolox/eslint-config-wolox)| Wolox eslint standard rules.
+|[eslint-config-wolox-react](https://github.com/Wolox/eslint-config-wolox-react)| Wolox eslint React rules.
+|[prettier]()| Code formatter.
+|[react-hot-loader](https://www.npmjs.com/package/react-hot-loader)| Allow live reloaded without loss of state.
+|[postcss](https://www.npmjs.com/package/postcss)| Is a tool for transforming styles with JS plugins.
+|[enzyme](https://www.npmjs.com/package/enzyme)| Is a JavaScript testing utility for React that makes it easier to test your React Components' output.
```

Comment:
`| JavaScript testing utility for React that makes it easier to test your React Components' output`